.github/workflows/manual-build.yaml

```diff
@@ -45,21 +45,40 @@ jobs:
         uses: docker/login-action@v1
         with:
           registry: quay.io
-          username: ${{ secrets.QUAY_LVMO_ROBOT_NAME_2 }}
-          password: ${{ secrets.QUAY_LVMO_ROBOT_TOKEN_2 }}
+          username: ${{ secrets.QUAY_LVMO_ROBOT_NAME }}
```

Comment:
The changes look good to me, I think we should use `registry_namespace` here instead of the `secrets`. Because if someone gives values other than lvmo we won't be able to push the images though login will succeed. I would like it to fail earlier in that case at login itself. Also we can move login a bit up in the steps before go cache stuff.

Reply:
Fixed.